src/migrate/arcgis/index.md

```diff
@@ -0,0 +1,96 @@
+
+Alternatively, you can convert the data files you use in the project to [supported](../../gis/supported_formats.md) formats and recreate the project and its settings in QGIS manually.
+
+To use your QGIS project with the <MainPlatformNameLink /> platform:
+1. [Sign up to <MainPlatformName />](../../setup/sign-up-to-mergin-maps/)
+2. [Install the <QGISPluginName />](../../setup/install-mergin-maps-plugin-for-qgis/)
+3. [Install the <MobileAppName />](../../setup/install-mobile-app/)
+4. Sync the QGIS project to <MobileAppName /> using the <QGISPluginNameShort />. See how the settings done in QGIS translate to the <MobileAppNameShort />.
+
+### SLYR 
+
+The [SLYR tool](https://north-road.com/slyr/) by [North Road](https://north-road.com) facilitates the migration from ArcGIS to QGIS by enabling the conversion of various proprietary data formats, styles and project files to QGIS-compatible formats, such as the conversion of MXD (ArcMap) and AXL (<NoSpellcheck id="ArcIMS" />) files to QGIS project files.
+
+This tool helps users transition smoothly by preserving complex symbology, layouts, and project structure. It simplifies the shift from Esri tools to the open-source QGIS environment, reducing the time and effort needed to replicate existing workflows. 
+
+SLYR tool offers Community and Licensed version:
+- Community version is open-sourced on <GitHubRepo id="north-road/slyr" desc="GitHub" /> and can be freely installed via QGIS Plugin Manager to extract, parse, and convert Esri `.lyr` and `.style` files to QGIS-compatible formats. 
+- Licensed version is closed-sourced and has features such as converting `MXD/MXT` project files to `QGS` or converting various data formats to GeoPackage. 
+
+Full comparison of the version can be found on North Road's [SLYR project page](https://north-road.com/slyr/). 
+
+![SLYR logo](./slyr.png "SLYR logo")
+
+
+### Domains 
+
+ArcGIS Pro [domains](https://pro.arcgis.com/en/pro-app/latest/help/data/geodatabases/overview/create-modify-and-delete-domains.htm) are used to restrict the valid values allowed in an attribute field. Convert them manually to QGIS drop-down widgets - likely [Value Relation](../../layer/form-widgets) - could be time consuming. You can export the domains to CSV to be imported and set value relation widgets, along with it's custom expression visibility to mimic cascading forms. We again recommend using [SLYR](#slyr) to facilitate the conversion of domains to QGIS.
+
+### Data Formats
+
+ArcGIS Pro [supports GeoPackages](https://pro.arcgis.com/en/pro-app/latest/help/data/databases/work-with-sqlite-databases-in-arcgis-pro.htm), which are the primary and recommended format for your survey layer in <MainPlatformName />. Both ArcGIS and QGIS can also handle Shapefile, GeoTIFF, GeoJSON, WMS, WFS, and PostGIS layers. However, we do not recommend using [Shapefile](http://switchfromshapefile.org) in <MainPlatformName />, as we do not support collaborative use for this format.
```

Comment:
ArcGIS Pro is **really** finicky about geopackage. It works ok sometimes, other times for no discernable reason it will refuse to open a gpkg. 

I would reword to this say "ArcGIS Pro has limited support for Geopackage"